cms/templates/certificates.html

```diff
@@ -67,10 +67,15 @@ <h2 class="sr title">${_("Certificates")}</h2>
         <div class="bit">
           <div class="certificates-doc">
             <h2 class="title-3">${_("Certificates")}</h2>
-            <p>${_("Upon successful completion of your course, learners receive a certificate to acknowledge their accomplishment. If you are a course team member with the Admin role in Studio, you can configure your course certificate.")}</p>
-            <p>${_("Click {em_start}Add your first certificate{em_end} to add a certificate configuration. Upload the organization logo to be used on the certificate, and specify at least one signatory. You can include up to four signatories for a certificate. You can also upload a signature image file for each signatory. {em_start}Note:{em_end} Signature images are used only for verified certificates. Optionally, specify a different course title to use on your course certificate. You might want to use a different title if, for example, the official course name is too long to display well on a certificate.") .format(em_start='<strong>', em_end="</strong>")}</p>
-            <p>${_("Select a course mode and click {em_start}Preview Certificate{em_end} to preview the certificate that a learner in the selected enrollment track would receive. When the certificate is ready for issuing, click {em_start}Activate.{em_end} To stop issuing an active certificate, click {em_start}Deactivate{em_end}.").format(em_start='<strong>', em_end="</strong>")}</p>
-            <p>${_(" To edit the certificate configuration, hover over the top right corner of the form and click {em_start}Edit{em_end}. To delete a certificate, hover over the top right corner of the form and click the delete icon. In general, do not delete certificates after a course has started, because some certificates might already have been issued to learners.").format(em_start="<strong>", em_end="</strong>")}</p>
+            <h3 class="title-3">${_("Working with Certificates")}</h3>
+            <p>${_("Specify a course title to use on the certificate if the course's official title is too long to be displayed well.")}</p>
+            <p>${_("For verified certificates, upload between one and four signatory images.")}</p>
+            <p>${_("To edit or delete an certificate before it is activated, hover over the top right corner of the form and select {em_start}Edit{em_end} or the delete icon.").format(em_start="<strong>", em_end="</strong>")}</p>
+            <p>${_("To test a certificate, select a course mode and select {em_start}Preview Certificate{em_end}.").format(em_start='<strong>', em_end="</strong>")}</p>
```

Comment:
I think this is "previewing" a certificate more than actually testing it. Maybe "view" so that you don't have to repeat "preview": "To view a certificate, select a course mode..."